contracts/schemes/ContinuousLockingToken4Reputation.sol

```diff
@@ -67,16 +67,19 @@ contract ContinuousLocking4Reputation is Agreement {
      * @param _avatar the avatar to mint reputation from
      * @param _reputationReward the reputation reward per locking batch that this contract will reward
```

Comment:
This value is assinged to _reputationRewardLeft, which is a global  variable, so (it seems to me) this is not the reputatino reward per locking batch, butthe TOTAL amount of reputation that can be minted by this contract.